tests/js/spec/views/__snapshots__/installWizard.spec.jsx.snap

```diff
@@ -35,7 +35,20 @@ exports[`InstallWizard renders 1`] = `
             errorMessage="Unable to save your changes. Please ensure all fields are valid and try again."
             footerClass="form-actions align-right"
             hideErrors={false}
-            initialData={Object {}}
+            initialData={
+              Object {
+                "auth.allow-registration": undefined,
```

Comment:
Is this right?

Reply:
Let me run one more QA pass to make sure all the values are being respected as we think they should be.

Reply:
everything appears to work as expected when testing by hand so the best I can say is "yes"

Reply:
actually i lied - something on /manage/ settings is causing the form to re-render so need to debug whats not saving correctly

Reply:
there's an issue with the defaults not saving in a way we want, so just hacked around the specific option which needs this behavior

Reply:
not sure why this snapshot is still the same tbqh... id have expected no changes other than something to beacon.anonymous